integration-tests/src/test/java/com/datastax/oss/driver/api/core/ConnectIT.java

```diff
@@ -98,6 +100,34 @@ public void should_wait_for_contact_points_if_reconnection_enabled() throws Exce
     session.close();
   }
 
+  @Test
+  public void should_cleanup_on_lbp_init_failure() throws Exception {
```

Comment:
Please add a little note in the javadocs of this method to link this test to JAVA-1948 for future reference.